packages/launcher/package.json

```diff
@@ -4,19 +4,15 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "build": "bin-up tsc --project .",
-    "build-js": "bin-up tsc --project .",
-    "build-prod": "npm run build",
-    "check-deps": "node ../../scripts/check-deps.js --verbose",
-    "check-deps-pre": "npm run check-deps -- --prescript",
-    "preclean": "npm run check-deps-pre",
+    "build": "tsc --project .",
+    "build-js": "tsc --project .",
+    "build-prod": "yarn build",
     "clean": "node scripts/clean.js || true",
     "clean-deps": "rm -rf node_modules",
-    "clean-js": "npm run clean",
-    "size": "npm pack --dry",
-    "pretest": "npm run check-deps-pre",
-    "test": "npm run unit",
-    "unit": "bin-up mocha --reporter mocha-multi-reporters --reporter-options configFile=../../mocha-reporter-config.json"
+    "clean-js": "yarn clean",
+    "size": "t=\"$(npm pack .)\"; wc -c \"${t}\"; tar tvf \"${t}\"; rm \"${t}\";",
```

Comment:
I think it should be `$(yarn pack)` instead of `$(yarn pack .)`, but I need to confirm this. I'll update this reference to use yarn and test with the `cli` package as well.